app/templates/records/macros/related_records.html

```diff
@@ -0,0 +1,62 @@
+{% macro related_records(record) %}
```

Comment:
Is this macro used?

Reply:
removed duplicate file as this is not used.